client/blocks/taxonomy-manager/list-item.jsx

```diff
@@ -157,18 +161,14 @@ class TaxonomyManagerListItem extends Component {
 				{ typeof term.post_count !== 'undefined' && (
 					<div className="taxonomy-manager__count">
 						<Count
-							ref="count"
+							ref={ this.countRef }
 							count={ term.post_count }
 							onMouseEnter={ this.showTooltip }
 							onMouseLeave={ this.hideTooltip }
 						/>
 					</div>
 				) }
-				<Tooltip
-					context={ this.refs && this.refs.count }
-					isVisible={ this.state.showTooltip }
-					position="left"
-				>
+				<Tooltip context={ this.countRef } isVisible={ this.state.showTooltip } position="left">
```

Comment:
Needs to be `this.countRef.current` to pass the element, not the ref object.

Reply:
Good tip, I was incorrectly assuming that the ref object had the same format before and after.